addons/mail/views/mail_template_views.xml

```diff
@@ -96,33 +93,33 @@
             <field name="name">email.template.tree</field>
             <field name="model">mail.template</field>
             <field name="arch" type="xml">
-                <tree string="Templates">
+                <tree string="Templates" expand="1">
```

Comment:
While being here, maybe rename to "mail.template.view.tree" :) will make some people happy